src/main/java/stsjorbsmod/actions/MemoryManagerSnapAction.java

```diff
@@ -0,0 +1,18 @@
+package stsjorbsmod.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import stsjorbsmod.memories.MemoryManager;
+
+public class MemoryManagerSnapAction extends AbstractGameAction {
```

Comment:
Was this intended to be part of this PR? (is it necessary at all? normally you wouldn't want to invoke snap outside of the fuller SnapAction that already exists)

Reply:
nope, I thought I had backed stuff out that i was playing around with. but clearly not.